.changeset/real-kings-hide.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
We don't need to include this PR in the changelog since its just an internal refactor